deebot_client/hardware/deebot/55aiho.py

```diff
@@ -0,0 +1 @@
+x5d34r.py
```

Comment:
Please create a symbol link to the `x5d34r.py` file. It looks like this is a text file with the name in it